src/OSPSuite.UI/Views/Importer/ColumnMappingControl.cs

```diff
@@ -194,12 +205,14 @@ private RepositoryItem lloqRepository(ColumnMappingDTO model)
          {
             return _lloqButtonRepository;
          }
+
          return _disabledLloqButtonRepository;
       }
 
       public void Rebind()
-      { 
-         _gridViewBinder.Rebind();
+      {
+         columnMappingGridView.RefreshData();
+         // _gridViewBinder.Rebind();
```

Comment:
We need to discuss here. Rebind is called quite often. It's required when adding or deleting rows. But when modifying bound date, it is not